### ISSUE TITLE
Add guide to codesigning and running on a real device

### ISSUE THREAD

**Issue by kubkon** [kubkon/zig-ios-example#3]
It'd be cool if the README included a guide to codesigning and running the generated app on a real device using `codesign` and `xcrun` tools (or otherwise).

@jmrico01 would you be willing to add such a short guide to the README?

**Comment by jmrico01** [comment]
For sure! I'll write something and PR it here when I get the chance. Working on my Zig app at the moment :)

**Comment by MaxLeiter** [comment]
Is there an update on this? I'm interested in using Zig to build an iOS app.

**Comment by jmrico01** [comment]
Hey! Sorry I stopped working on my iOS project for a bit, but this is good timing because I've been meaning to revisit. I'll try to add something to the readme in the next day or two - feel free to ping me if I forget.

**Comment by jmrico01** [comment]
Aha, seems like @kubkon made some progress while I was away. I had some very rough setup for code signing and hadn't really figured out programmatic deployment to an iOS device, so I am checking out https://github.com/kubkon/zignature and https://github.com/kubkon/zig-deploy, tweaking some outdated things, and seeing if I can make them work.

**Comment by kubkon** [comment]
> Aha, seems like @kubkon made some progress while I was away. I had some very rough setup for code signing and hadn't really figured out programmatic deployment to an iOS device, so I am checking out https://github.com/kubkon/zignature and https://github.com/kubkon/zig-deploy, tweaking some outdated things, and seeing if I can make them work.

Yes I did :-) In order to make this example work fully programmatically with `zignature` and `zig-deploy` and with any valid and Apple issued developer certificate, there's quite a bit of work left, and in particular:
* `zignature` needs to learn how to parse XML to learn about bundle's metadata
* `zignature` needs to learn how to DER encode
* `zig-deploy` needs more error handling and testing as currently it mainly handles non-incremental installs and doesn't check if app already exists on the target device or not. It also doesn't install any developer profiles on the device.

Anyhow, here's what the entire workflow should look like in the end and what I am aiming for: [iOS DX with Zig: demo at Zig Milan meetup](http://www.jakubkonka.com/2022/04/30/ios-dx-with-zig-milan-meetup.html).
